scripts/csi-driver-smoke/run-test.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
IMHO this should be in a different repo, either directly in the csi-driver repo or a related one.

Reply:
The actual test runner script is in the csi-driver repo. This script just checks-out the csi-driver repo and the relevant PR and then runs the test runner from the checkout.